.circleci/config.yml

```diff
@@ -168,6 +173,11 @@ caffe2_linux_build_defaults: &caffe2_linux_build_defaults
           cmake_args+=("-DBUILD_ATEN=ON")
         fi
 
+        if [[ $BUILD_ENVIRONMENT == py2-cuda9.0-cudnn7-ubuntu16.04 ]]; then
+          cmake_args+=("-DUSE_TENSORRT=ON")
```

Comment:
Almost there! This line is ineffective because of https://github.com/pytorch/pytorch/blob/62f4db6d8a5c72621e8a88c9f0144d98225dea7c/.jenkins/caffe2/build.sh#L11

We should put it here:
https://github.com/pytorch/pytorch/blob/62f4db6d8a5c72621e8a88c9f0144d98225dea7c/.jenkins/caffe2/build.sh#L125

Reply:
Ah. Thanks! Done.